src/cp_control_types.F

```diff
@@ -512,6 +512,12 @@ MODULE cp_control_types
       !> user-defined reference point
       REAL(kind=dp), DIMENSION(:), POINTER :: dipole_ref_point
       !
+      ! SOC subsection
+      ! Number of Singlett states to be used
+      INTEGER                              :: soc_singlett
+      ! Number of triplett states to be used
+      INTEGER                              :: soc_triplett
+      !
```

Comment:
soc_singlet soc_triplet

Reply:
Thank you. This error should be resolved now.